interp/constrintern.ml

```diff
@@ -865,6 +865,8 @@ let intern_qualid loc qid intern env lvar us args =
           DAst.make ?loc @@ GApp (DAst.make ?loc:loc' @@ GRef (ref, us), arg)
         | _ -> err ()
         end
+      | Some [Misctypes.GSet], GSort (Misctypes.GType _) -> DAst.make ?loc @@ GSort Misctypes.GSet
+      | Some [Misctypes.GProp], GSort (Misctypes.GType _) -> DAst.make ?loc @@ GSort Misctypes.GProp
```

Comment:
You probably only want to handle `GSort (GType [])` on the right, ie Type without explicit sort. Otherwise something like
``` coq
Universe u.
Notation foo := Type@{u}.
Check foo@{Set}.
```
would produce `Set` which is strange, it should be an error.

Also you probably want to handle `foo@{v}` which means you would do
```ocaml
| Some [s], GSort (Misctypes.GType _) -> DAst.make ?loc @@ GSort (glob_sort_of_level s)
```
defining on the side `glob_sort_of_level` translating a `glob_level` to a `glob_sort`.

Reply:
Ping.
Also let me use the github request change thingy, see above comment.